### ISSUE TITLE
"distance" and "totalDistance" attributes

### ISSUE THREAD

**Issue by Fleetracking** [tananaev/traccar#2657]
Hi 
After an upgrade from 3.3 to 3.8 version (i use vitalidze GUI), "distance" and "totalDistance" attributes are stored at Mysql DB even if "distance.enable" parameter is not configured at "traccar.xml" and "default.xml" config files.
Do you have an idea regarding this issue? i didn't have an answer from Vitalidze forum (cf reported [issue](https://github.com/vitalidze/traccar-web/issues/973)).
BR

**Comment by Abyss777** [comment]
Hello,
It is an expected behavior. Distance calculation is always enabled and can't be switched off.

**Comment by Fleetracking** [comment]
Hi
It isn't the case for previous version.
But what about "totalDistance" ? it's not enabled anywhere in my traccar conf files.

**Comment by Abyss777** [comment]
I meant both are always enabled.

**Comment by Fleetracking** [comment]
I dont think so.
No distance attributes for my all recorded Mysql data before upgrade.
And at this case what's the effect of "distance.enable" parameter.
BR

**Comment by Abyss777** [comment]
I don't understand your question.
Again:
Distance calculation is always enabled and can't be disabled since 3.7 version.
It does not mean that old data (before upgrade) was upgraded. It means that all new positions (since upgrade to 3.7) will have `distance` and `totalDistance` attribute.

**Comment by Fleetracking** [comment]
I get this point, because as mentioned i have before 3.3 version.
My question is relative to `distance.enable` parameter (ref to [traccar.xml](https://www.traccar.org/configuration-file/) conf file)
What's the effect of this command?
BR

**Comment by Abyss777** [comment]
There is no effect, it is ignored (not used).

**Comment by Fleetracking** [comment]
OK, so since 3.7 version no effect of this parameter

**Comment by wolfbgn** [comment]
Is it possible to display "distance" and "totalDistance" attributes in kilomiters?

**Comment by tananaev** [comment]
Yes, you just need to select distance units preference in the user or server settings.

**Comment by wolfbgn** [comment]
Thanks.
